src/sentry/web/frontend/pipeline_advancer.py

```diff
@@ -40,9 +41,12 @@ def handle(self, request, provider_id):
             and request.GET.get("setup_action") == "install"
             and pipeline is None
         ):
-            installation_id = request.GET.get("installation_id")
+            signed_params = sign(
+                installation_id=request.GET.get("installation_id"),
+                provider_id=provider_id,
+            )
             return self.redirect(
-                reverse("integration-installation", args=[provider_id, installation_id])
+                f"{reverse('github-integration-installation')}/?signed_params={signed_params}"
```

Comment:
@AniketDas-Tekky might want to make a comment how Github is the only integration to use this which is why it's Github specific